setup.py

```diff
@@ -19,7 +19,7 @@
     },
     install_requires=[
         'argparse==1.3.0',
-        'PyGithub==1.25.2',
+        'PyGithub==1.25.2-squash',
```

Comment:
@babineaum is `PyGithub==1.25.2-squash` published to public pypi from your fork?